allensdk/brain_observatory/behavior/behavior_ophys_session.py

```diff
@@ -1,98 +1,203 @@
 import numpy as np
 import pandas as pd
 import xarray as xr
-import math
-from typing import NamedTuple
-import os
+from typing import Any
 
-from allensdk.core.lazy_property import LazyProperty, LazyPropertyMixin
 from allensdk.internal.api.behavior_ophys_api import BehaviorOphysLimsApi
-from allensdk.brain_observatory.behavior.behavior_ophys_api.behavior_ophys_nwb_api import equals, BehaviorOphysNwbApi
+from allensdk.brain_observatory.behavior.behavior_ophys_api\
+    .behavior_ophys_nwb_api import BehaviorOphysNwbApi
 from allensdk.deprecated import legacy
-from allensdk.brain_observatory.behavior.trials_processing import calculate_reward_rate
-from allensdk.brain_observatory.behavior.dprime import get_rolling_dprime, get_trial_count_corrected_false_alarm_rate, get_trial_count_corrected_hit_rate
-from allensdk.brain_observatory.behavior.dprime import get_hit_rate, get_false_alarm_rate
+from allensdk.brain_observatory.behavior.trials_processing\
+    import calculate_reward_rate
+from allensdk.brain_observatory.behavior.dprime import\
+    get_rolling_dprime, get_trial_count_corrected_false_alarm_rate,\
+    get_trial_count_corrected_hit_rate
```

Comment:
Maybe something like?

```
from allensdk.brain_observatory.behavior.dprime import (
    get_rolling_dprime, get_trial_count_corrected_false_alarm_rate,
    get_trial_count_corrected_hit_rate)
```

Reply:
I'll do that for the long import lists. FYI there is still one absolute path that is too long for the character limit where I'll have to backslash.